pkg/manager/member/tikv_member_manager.go

```diff
@@ -1025,6 +1041,15 @@ func tikvStatefulSetIsUpgrading(podLister corelisters.PodLister, pdControl pdapi
 	return false, nil
 }
 
+// TODO: Support check pd status http request in future.
+func buildTiKVReadinessProbHandler() corev1.Handler {
```

Comment:
Similar to `buildTiDBReadinessProbHandler`, maybe we need to introduce an `*v1alpha1.TidbCluster` input parameter here for future use.